osm2gtfs/creators/routes_creator.py

```diff
@@ -13,4 +13,67 @@ def __repr__(self):
         return rep
 
     def add_routes_to_feed(self, feed, data):
-        raise NotImplementedError("Should have implemented this")
+        """
+        This function adds the routes from the data to the GTFS feed.
+        """
+        # Get route information
+        routes = data.get_routes()
+
+        # Loop through all routes
+        for route_ref, route in sorted(routes.iteritems()):
+
+            # Add route information
+            gtfs_route = feed.AddRoute(
+                route_id=self._define_route_id(route),
+                route_type=self._define_route_type(route),
+                short_name=route.route_id.encode('utf-8'),
+                long_name=route.name
+            )
+            gtfs_route.agency_id = feed.GetDefaultAgency().agency_id
+            gtfs_route.route_desc = self._define_route_description(route)
+            gtfs_route.route_url = self._define_route_url(route)
+            gtfs_route.route_color = self._define_route_color(route)
+            gtfs_route.route_text_color = self._define_route_text_color(route)
+        return
+
+    def _define_route_id(self, route):
+        """
+        Returns the route_id for the use in the GTFS feed.
+        Can be easily overridden in any creator.
```

Comment:
I think we don't need to state "Can be easily overridden in any creator." in every method, that should be clear.

Reply:
I put it there on purpose, because it is the reason to define those standard creator functions. Sure, we could take them out. For me I like them, because I hope they help to explain the code better. Even if people look only at one function (on standard creator they want to implement) and not the whole application.

Reply:
I agree with @xamanu

Reply:
OK, then leave it.